lib/views/home/paymentlist_view.dart

```diff
@@ -131,56 +140,62 @@ class _PaymentListViewState extends State<PaymentListView> {
                 shrinkWrap: true,
                 itemCount: snapshot.data!.length,
                 itemBuilder: (context, index) {
-                  return ExpandableCard(
-                    expandedAlignment: Alignment.topLeft,
-                    expandableChild: _buildSpecificPaymentView(
-                        context: context, items: snapshot.data!, index: index),
-                    child: SizedBox(
-                      height: MediaQuery.of(context).size.height * 0.1,
-                      width: MediaQuery.of(context).size.width,
-                      child: Row(
-                        children: [
-                          Container(
-                            height: 46,
-                            width: 46,
-                            decoration: const BoxDecoration(
-                              shape: BoxShape.circle,
-                            ),
-                            alignment: Alignment.center,
-                            child: checkListIdentifier(
-                                    listTile: snapshot.data![index])
-                                ? const Icon(
-                                    Icons.arrow_downward,
-                                    color: green,
-                                  )
-                                : const Icon(
-                                    Icons.arrow_upward,
-                                    color: red,
-                                  ),
-                          ),
-                          Expanded(
-                            child: Container(
+                  return Padding(
+                    padding: EdgeInsets.symmetric(
+                        horizontal: isScreenWidthLarge(size) ? 200 : 0),
```

Comment:
this padding is useful in most of our component, also in the keyboard!

Do you think that we can abstract this inside a `PadContainer` component inside the trash component?

So we can use this like

```dart
 PadContainer(
  padding: Platform.isScreenWidthLarge(size) ? 200 : 0,
  children: ....,
)
```